src/tests/nssdb.rs

```diff
@@ -175,6 +175,9 @@ fn test_nssdb_token() {
     let ret = fn_find_objects_final(session);
     assert_eq!(ret, CKR_OK);
 
+    #[cfg(not(feature = "rsa"))]
```

Comment:
Given nssdb always requires HMAC, what about changing some of these operations to use/generate a SECRET value usable by HMAC and defer RSA testing to the general RSA testing?

RSA is not really key here, what is ey is testing that encryption.decryption of a secret attribute works and fetching works as expected.

Reply:
Indeed, I can do that so we can move forward with this change.

Reply:
Up to you.

Reply:
I will drop it then. The current solution with if feature + return is ugly.

Reply:
Or try to move it to RSA test if possible.